 If LCLType is not included, then THandle is System.THandle
 rather than LCLType.THandle. System.THandle is 32 bits wide.

--- lazarus/lcl/interfaces/gtk/gtkwsdialogs.pp
+++ lazarus/lcl/interfaces/gtk/gtkwsdialogs.pp
@@ -29,7 +29,7 @@
 uses
   {$IFDEF GTK2} Gtk2, Glib2, gdk2, {$ELSE} Gtk, gdk, Glib, {$ENDIF}
   SysUtils, Classes, Controls, LMessages, InterfaceBase, graphics,
-  Dialogs, WSDialogs, WSLCLClasses, gtkint, gtkproc, gtkwscontrols;
+  LCLType, Dialogs, WSDialogs, WSLCLClasses, gtkint, gtkproc, gtkwscontrols;
 
 type
 
--- lazarus/lcl/widgetset/wsdialogs.pp
+++ lazarus/lcl/widgetset/wsdialogs.pp
@@ -44,7 +44,7 @@
 // To get as little as posible circles,
 // uncomment only when needed for registration
 ////////////////////////////////////////////////////
-  Dialogs,
+  LCLType, Dialogs,
 ////////////////////////////////////////////////////
   WSLCLClasses, WSControls;
 
